lib/srv/db/ca.go

```diff
@@ -103,16 +104,30 @@ func (s *Server) getCACert(ctx context.Context, database types.Database) ([]byte
 // getCACertPath returns the path where automatically downloaded root certificate
 // for the provided database is stored in the filesystem.
 func (s *Server) getCACertPath(database types.Database) (string, error) {
-	// All RDS and Redshift instances share the same root CA which can be
-	// downloaded from a well-known URL (sometimes region-specific). Each
-	// Cloud SQL instance has its own CA.
 	switch database.GetType() {
+	// All RDS instances share the same root CA (per AWS region) which can be
+	// downloaded from a well-known URL.
 	case types.DatabaseTypeRDS:
 		return filepath.Join(s.cfg.DataDir, filepath.Base(rdsCAURLForDatabase(database))), nil
+
+	// All Redshift instances share the same root CA which can be downloaded
+	// from a well-known URL.
 	case types.DatabaseTypeRedshift:
 		return filepath.Join(s.cfg.DataDir, filepath.Base(redshiftCAURLForDatabase(database))), nil
+
+	// ElastiCache databases are signed with Amazon root CA. In most cases,
+	// x509.SystemCertPool should be sufficient to verify ElastiCache servers.
+	// However, x509.SystemCertPool does not support windows for go versions
+	// older than 1.18. In addition, system cert path can be overridden by
+	// environment variables on many OSes. Therefore, Amazon root CA is
+	// downloaded here to be safe.
+	case types.DatabaseTypeElastiCache:
```

Comment:
This is probably not related to this PR only, but do we have any logic to update certs when needed? EC2 allows you to update the system certs that solves the problem. If we manage the CA, then we also need some way of updating them.

Reply:
no we don't manage it well. users have to manually remove the files. I've created some tickets to track those before.
https://github.com/gravitational/teleport/issues/9909
https://github.com/gravitational/teleport/issues/9910